media/src/androidMain/kotlin/dev/icerock/moko/media/picker/MediaPickerFragment.kt

```diff
@@ -45,6 +45,7 @@ class MediaPickerFragment : Fragment() {
         startActivityForResult(intent, requestCode)
     }
 
+    @Suppress("UnreachableCode")
```

Comment:
why itis unreachable?

Reply:
I don't know why, but detect is sure that all code inside onActivityResult is unreachable.
I checked if it actually works, and it works
So I add a Suppress annotation